setup.cfg

```diff
@@ -0,0 +1,27 @@
+[metadata]
+name = pre_commit_all_the_way_down
+version = 0.0.1
+description = Apply pre-commit hooks on chunks of files as a pre-commit hook.
+long_description = file: Readme.md
+long_description_content_type = text/markdown
+url = https://github.com/cphyc/pre-commit-all-the-way-down
+license = MIT
+classifiers =
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+authors = Corentin Cadiou <contact@cphyc.me>
+
+[options]
+py_modules = pre_commit_all_the_way_down
+python_requires = >=3.6.1
+
+[options.entry_points]
+console_scripts =
+    pre-commit-all-the-way-down=pre_commit_all_the_way_down:main
```

Comment:
obviously this is no doing anything at the moment, but do you intend to actually have a console script here and what for ?

Reply:
I had the (maybe wrong) impression it was required, or at least would simplify, calling it as a hook. This way, the `entry` in the `.pre-commit-hooks.yaml` file can simply be `pre-commit-all-the-way-down`

Reply:
well it's not *required* but it is certainly the easiest way to do it with a hook that's written in Python.
But my remark from the other PR also applies here: I don't think any actual program/hook should bear the name "pre-commit-all-the-way-down". If the one you're implementing in #1 is named "python-doc", it should also be the name of the entry point (as well as filename).

Reply:
Fair enough. Changing it to `python-doc`.